alembic/versions/392e49db40fc_add_sample_concentration_min_max.py

```diff
@@ -0,0 +1,38 @@
+"""Add sample concentration min max
+
+Revision ID: 392e49db40fc
+Revises: db61c62d9bc0
+Create Date: 2023-10-25 09:50:16.779186
+
+"""
+import sqlalchemy as sa
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "392e49db40fc"
+down_revision = "db61c62d9bc0"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        table_name="application",
+        column=sa.Column(
+            "sample_concentration_minimum",
```

Comment:
Where do we standardise the unit? IN the web interface and docs?

Reply:
Good point - would it be preferable to have sample_concentration_minimum_ng_ul here? Or maybe one can fix the web interface to add that as a suffix? @henrikstranneheim

Reply:
I think both sounds good, to specify it in the fe and in the naming of the column here

Reply:
👍

Reply:
Would a good compromise be to append the unit in the FE and keep the column name without unit? Sort of like this but without units in the column name?
<img width="263" alt="image" src="https://github.com/Clinical-Genomics/cg/assets/40887124/16401146-eb0f-4dd9-be0c-a2ee44e8bd1d">

Reply:
So here the FE appends the units, whereas the DB only holds the number values. Or will some people go directly to statusdb and not use the web interface and wonder what units are used?

Reply:
Hm, yeah it probably does not make sense to include the units in the column name in the db. Sounds good 👍